### ISSUE TITLE
Add the ability to disable client heartbeat

### ISSUE THREAD

**Issue by aofei** [juicedata/juicefs#3329]
**What would you like to be added**:

Please consider making `juicefs mount --heartbeat 0 ...` to disable the client heartbeat feature.

**Why is this needed**:

My use case:

```bash
$ juicefs format \
	--storage sqlite3 \
	--bucket /litestream/data-dir.jfs-bucket.db \
	sqlite3:///litestream/data-dir.jfs-meta.db \
	data-dir
$ litestream replicate -sync-interval 1 /litestream/data-dir.jfs-meta.db s3://mybkt/data-dir.jfs-meta.db &
$ litestream replicate -sync-interval 1 /litestream/data-dir.jfs-bucket.db s3://mybkt/data-dir.jfs-bucket.db &
$ juicefs mount sqlite3:///litestream/data-dir.jfs-meta.db /data
```

The files in `/data` rarely change, but I still expect JuiceFS to reflect to those two databases when it does, and [Litestream](https://litestream.io) to stream WALs to my S3 bucket in time. They actually do work pretty well together. Until I got my S3 bill.

Again, the files in `/data` are rarely change, so I'd like JuiceFS to be quiet until it does, which means it shouldn't be doing any writes to those two databases. But here comes the problem. It turns out that there is a heartbeat feature for the client here. The client always send heartbeats to meta storage at regular intervals (which defaults to 12s, and fixed in the [1, 60] interval). This means that by default every 12s Litestream will stream two WALs to my S3 bucket (I don't know why two). In other words, two S3 class A operations every 12s, or 432,000 class A operations every month, for nothing. And I have hundreds of the above setups currently active...

I know that for network meta storage, such as Redis, the client heartbeat feature is indispensable. But when it comes to SQLite meta storage, I believe this feature is unnecessary, given that there is only local disk I/O.

So I think it's reasonable to add the ability to disable the client heartbeat.

**Comment by solracsf** [comment]
Applies also to BadgerDB.

**Comment by aofei** [comment]
Hi @solracsf! And yes, I'm aware of the `--heartbeat` option. But, like I said, `--heartbeat 0` is actually equivalent to `--heartbeat 1`.

> ... which defaults to 12s, and fixed in the [1, 60] interval...

**Comment by solracsf** [comment]
True
https://github.com/juicedata/juicefs/blob/c123b05cd03c1a791bd334a67232f0d3cff251df/pkg/meta/config.go#L57-L59
Indeed, either for SQLite and BadgerDB `0`  should be allowed.

**Comment by SandyXSD** [comment]
Fixed by #3471

**Comment by SandyXSD** [comment]
There is still some background updates with low frequency per volume, e.g. https://github.com/juicedata/juicefs/blob/f1c475d9f7b543176d1123567dee85245450de8b/pkg/meta/base.go#L424

**Comment by aofei** [comment]
> Fixed by #3471

Thanks for the fix! ❤️

> There is still some background updates with low frequency per volume, e.g.
> 
> https://github.com/juicedata/juicefs/blob/f1c475d9f7b543176d1123567dee85245450de8b/pkg/meta/base.go#L424

I'd take that as acceptable, given that `--no-bgjob` is already there.

I'm going to close this issue.
